llvm/include/llvm/BitstreamCAS/BitstreamCASNode.h

```diff
@@ -0,0 +1,13 @@
+#include "llvm/ADT/SmallVector.h"
```

Comment:
Nit: since this is a small struct in a file with no license header, I will just move this structure into `BitstreamCASWriter.h`.